src/Data/Fin/Base.agda

```diff
@@ -95,9 +93,9 @@ zero    ↑ʳ i = i
 
 -- reduce≥ "m + i" _ = "i".
 
-reduce≥ : ∀ (i : Fin (m ℕ.+ n)) (i≥m : toℕ i ℕ.≥ m) → Fin n
-reduce≥ {zero}  i       i≥m       = i
-reduce≥ {suc m} (suc i) (s≤s i≥m) = reduce≥ i i≥m
+reduce≥ : ∀ (i : Fin (m ℕ.+ n)) → .(m ℕ.≤ (toℕ i)) → Fin n
+reduce≥ {zero}  i       _   = i
+reduce≥ {suc m} (suc i) m≤i = reduce≥ {m} i (ℕ.s≤s⁻¹ m≤i)
```

Comment:
Is the application of `{m}` really necessary here? A bit worried if it is...

Reply:
Probably not. More 'note-to-self/others' internal documentation... but we've disagreed before about my tendency to overwork the implicit argument inference, so sometimes I put things back which aren't perhaps necessary? Anyway, I didn't write the original definition, so it's probably legacy! Will fix...